wiki/People/The_Team/Beatmap_Nominators/Rules/tr.md

```diff
@@ -0,0 +1,70 @@
+
+## Aday gösterme sıfırlama
+
+- **Bir beatmapin aday göstermesi/göstermeleri sıfırlandığında, gönderinin ilk satırı sıfırlamanın kısa bir gerekçesini içermelidir.** Gönderileri linklemek yeterli değildir, alakalı gönderileri özetlemeyi denemelisiniz. Diskalifikasyon durumlarında, sadece bunun talep edildiğini yazmak da yeterli değildir. Gönderilecek çok fazla sorun olmadığı sürece beatmap tartışmalarında gönderilen her bir alakalı sorunu linklemelisiniz.
+- **Eğer aday gösterilen bir beatmap derecelendirilmez sorunlar içeriyorsa, onun aday göstermesi/göstermeleri sıfırlanmalıdır.** Beatmap setini aday göstermeye niyetlenmeseniz de bu geçerlidir.
+- **Beatmapin aday göstermesini/göstermelerini sıfırlamadan önce beatmapin en güncel versiyonuna bakıyor olduğunuzdan emin olun.**
+
+### Diskalifikasyon
+
+- **Sadece asil üyeler beatmapleri diskalifiye edebilir, sebebi ne olursa olsun.** Deneme sürecindeki üyeler diğer herkes gibi bu tarz tartışmalarda yer alabilir ve beatmapleri raporlayabilir.
+- **Diskalifikasyonlar temel olarak derecelendirilemez sorunlar içeren beatmapler için, ya da beatmap setinin sahibinin talebi üzerine uygulanır.**
+- **Beatmap seti sahibinin diskalifikasyon talepleri derhal yerine getirilebilir.** Kendi kendinizi diskalifiye etmek isterseniz, bu sizin için de geçerli.
+- **Çevrimiçi offsetler, etiketler vb. ile hafifletilebilen sorunlar da yine diskalifiye edilmelidir.** Bunun mümkün olmadığı durumlarda, bir NAT üyesiyle irtibata geçin. Çevrimiçi offsetler ve etiketler sadece websitesi için geçerlidir, dolayısıyla Dereceli olmadan önce düzeltilmediğinde oyuncular istemci içerisinde sorun yaşamaya devam edecektir.
+- **Derecelendirilemeyen ya da önemli sorunlar bulunan nitelikli beatmapler derhal diskalifiye edilmelidir.**
+- **Resmi olarak yeterliliğinizin bulunmadığı oyun modlarındaki beatmapleri diskalifiye etmeyin.** Bu, beatmapin oyun moduyla doğrudan alakası olmayan ses, arkaplan, veya diğer içeriklerle ilgili diskalifikasyonlar için geçerli değildir.
+
+### Veto
+
+Veto süreciyle ilgili tam bilgi için [beatmap vetosu](/wiki/People/The_Team/Beatmap_Nominators/Beatmap_Veto) sayfasına bakınız.
+
+- **Resmi olarak yeterliliğinizin bulunmadığı oyun modlarındaki beatmapleri veto etmeyin.** Bunları aday göstermeniz yasak dolayısıyla beatmapi yeniden aday gösterme yetkiniz olmayan bir aday göstermeyi bloklamak mantıksız. Bu, beatmapin oyun moduyla doğrudan alakası olmayan ses, arkaplan, veya diğer içeriklerle ilgili vetolar için geçerli değildir.
```

Comment:
the whole wiki page has been using geniş zaman and now we switch to şimdiki zaman?
```Bunları aday göstermeniz yasaktır, ve yeniden aday göstermeniz mümkün olmayan bir beatmapi aday göstermeyi bloklamak mantıksızdır [...]```